packages/docs/modules/page-config/blocks/api/index.vue

```diff
@@ -75,23 +82,28 @@ const eventsOptions = Object
   .entries(withManual.events || {})
   .filter(([key, prop]) => !prop.hidden)
   .map(([key, prop]) => ({
-    name: { name: key, ...prop },
+    name: key,
     description: t(getTranslation('events', key)),
   }))
```

Comment:
Why not to use `Array.prototype.reduce()` though? Instead of iterating over array two times, we do loop once (here and above).

Reply:
I would prefer to have readable code instead of reduce. In my opinion reduce must be used only for calculation in array, not to create new one.